### DISCUSSION TITLE
The v2 Update

### DISCUSSION THREAD

**Discussion by warengonzaga** [warengonzaga/buymeacoffee.js#22] [Announcements]
Hello, long time no update! Why? The team at Buy Me a Coffee is currently focused on shipping new features. That's why using their API at the moment is completely different and might cause bugs and issues. I would advise please wait for the official new set of APIs from them. From that point, that's my go signal to update this NodeJS client.

If you face any issues and problems while using this NodeJS client please let me know by reporting an issue.

**Comment by thomasbnt** [comment]
Can't wait for adding to projects. Some features requests for the future API is accepted to the team?

**Reply by warengonzaga** [reply]
Yes, we need to update the NodeJS client as well when the new version of APIs becomes publicly available.
I'm excited to implement BMC API to my projects as well just waiting for the new sets of APIs.
